STEER/STEERBase/AliGenHepMCEventHeader.cxx

```diff
@@ -96,7 +104,11 @@ AliGenHepMCEventHeader::AliGenHepMCEventHeader(
       Double_t var_x2,         // fraction of beam momentum carried by second parton ("target side")
       Double_t var_scalePDF,   // Q-scale used in evaluation of PDF's   (in GeV)
       Double_t var_pdf1,       // PDF (id1, x1, Q) - x*f(x)
-      Double_t var_pdf2        // PDF (id2, x2, Q) - x*f(x)
+      Double_t var_pdf2,       // PDF (id2, x2, Q) - x*f(x)
+      Float_t  sigma_gen,  // Cross section (for pp collisions)
+      Float_t  sigma_err,
```

Comment:
I would add a comment (even if it is quite clear).

Reply:
OK, I can add the comment.